shared/actions/fs/index.tsx

```diff
@@ -411,63 +411,54 @@ const cancelDownload = (action: FsGen.CancelDownloadPayload) =>
 const dismissDownload = (action: FsGen.DismissDownloadPayload) =>
   RPCTypes.SimpleFSSimpleFSDismissDownloadRpcPromise({downloadID: action.payload.downloadID})
 
-function* upload(_: Container.TypedState, action: FsGen.UploadPayload) {
-  const {parentPath, localPath} = action.payload
-  const opID = Constants.makeUUID()
-  const path = Constants.getUploadedPath(parentPath, localPath)
-  const src = {PathType: RPCTypes.PathType.local, local: Types.getNormalizedLocalPath(localPath)} as const
-
-  yield Saga.put(FsGen.createUploadStarted({path}))
-
-  // TODO: confirm overwrites?
```

Comment:
still could be nice - maybe ticket?

Reply:
The go side does that now! (only for files; for directories they might get merged)